eventdata/parameter_sources/randomevent.py

```diff
@@ -278,7 +280,20 @@ def generate_event(self):
         # set host name
         event["hostname"] = "web-{}-{}.elastic.co".format(event["geoip_continent_code"],random.randrange(1,3))
 
+        # determine the raw event size (as if this were contained in nginx log file. We do not bother to
+        # reformat the timestamp as this is not worth the overhead.
+        if self.record_raw_event_size:
+            raw_event = '%s - - [%s] "%s %s HTTP/%s" %s %s "%s" "%s"' % (event["clientip"], event["@timestamp"],
+                                                                         event["verb"], event["request"],
+                                                                         event["httpversion"], event["response"],
+                                                                         event["bytes"], event["referrer"],
+                                                                         event["agent"])
+            raw_event_size = len(raw_event)
+        else:
+            raw_event_size = 0
+
         line = '{"@timestamp": "%s", ' \
+               '"_raw_event_size":%d, ' \
```

Comment:
Similarly I am skeptical about having this new field (even with size `0`) for all existing benchmarks.
At the same time, I am aware if were to conditionally build the string depending on the flag, code would be messy and likely have a small perf hit.